{{cookiecutter.project_name}}/nix/sources.nix

```diff
@@ -12,36 +12,27 @@ let
     else
```

Comment:
Just an idea. We should be able to tell `cookiecutter` to run a script after the template is created, and we can run `niv init && niv update` there.

Reply:
Since we are setting the version of `niv` in the script I think it is safe to include `sources.nix` and `sources.json`. The other way around would require getting the `niv` executable. Even if we just call `nix-shell -p niv --run "niv init && niv update"` for the user, the `cookiecutter` script might have to run for quite some time.